core/src/test/scala/integration/kafka/api/SslAdminIntegrationTest.scala

```diff
@@ -42,6 +42,7 @@ object SslAdminIntegrationTest {
   @volatile var lastUpdateRequestContext: Option[AuthorizableRequestContext] = None
```

Comment:
Is there a reason we're not enabling KRaft on this file in this PR? I think it would make sense to do it together with BaseAdminIntegrationTest, SaslSslAdminIntegrationTest, especially since we're making changes to this file.

Reply:
The tests in this file test async Authorizer by using semaphore to block request threads. The functions used for this, don't work with KRaft at the moment (I get various failure from request not being blocked as expected or `This function can't be accessed when running the test in KRaft mode. ZooKeeper mode is required.` error). I think the changes required to enable KRaft  for this file, would be quite different than the other ones so I think it's better to be done in separate PRs. Also this was left out from the Jira issues created for the initial tests to be enabled with KRaft before 4.0, so I don't know if there was any particular reason for that.

Reply:
I created https://issues.apache.org/jira/browse/KAFKA-16974 for adding KRaft support to SslAdminIntegrationTest

Reply:
thanks @mimaison !